packages/form-js-viewer/src/render/components/util/sanitizerUtil.js

```diff
@@ -65,4 +68,60 @@ export function sanitizeMultiSelectValue(options) {
   }
 }
 
+/**
+ * Sanitizes an image source to ensure we only allow for data URI and links
+ * that start with http(s).
+ *
+ * Note: Most browsers anyway do not support script execution in <img> elements.
+ *
+ * @param {string} src
+ * @returns {string}
+ */
+export function sanitizeImageSource(src) {
+  const valid = ALLOWED_IMAGE_SRC_PATTERN.test(src);
+
+  return valid ? src : '';
+}
+
+/**
+ * Sanitizes an iframe source to ensure we only allow for links
+ * that start with http(s).
+ *
+ * @param {string} src
+ * @returns {string}
+ */
+export function sanitizeIFrameSource(src) {
+  const valid = ALLOWED_IFRAME_SRC_PATTERN.test(src);
+
+  return valid ? src : '';
+}
 
+/**
+ * Escapes HTML and returns pure text.
+ * @param {string} html
+ * @returns {string}
+ */
+export function escapeHTML(html) {
+  return html.replace(/[&<>"'{};:]/g, match => {
+    switch (match) {
```

Comment:
I thought we didn't use switches 😅

Reply:
Yeah, a map's better here anyways :)

Though an if else + would be insane :D